test/runtime/kafka.go

```diff
@@ -190,14 +187,8 @@ var _ = Describe("RuntimeValidatedKafka", func() {
 		Expect(endPoints[helpers.Enabled]).To(Equal(1), "Expected 1 endpoint to be policy enabled. Policy enforcement failed")
 		Expect(endPoints[helpers.Disabled]).To(Equal(2), "Expected 2 endpoint to be policy disabled. Policy enforcement failed")
 
-		// Waiting for kafka broker to be up.
-		err = waitForKafkaBroker(client, produceCmd)
-		Expect(err).To(BeNil(), "Kafka broker failed to come up")
-
-		By("By sending produce/consume request on topic `allowedTopic`")
-
 		By("Sending produce request on kafka topic `allowedTopic`")
-		for i := 1; i <= MaxMessages-1; i++ {
+		for i := 1; i <= MaxMessages; i++ {
```

Comment:
Can you explain the change for this conditional check?

Reply:
Earlier we were using the first produce request as a way to check the broker as well. Hence we only produced 1 to  MaxMessages - 1. Now this check was changed to test `createTopics` instead and hence we are producing all messages, i.e MaxMessages